lib/models/chat_history.dart

```diff
@@ -223,6 +223,41 @@ Stream<DeltaEvent> _handleDocumentChange(
           pinned: false);
       yield AppendDeltaEvent(model);
       break;
+    case "channel.poll.begin":
+      final model = TwitchPollEventModel(
```

Comment:
Move this to the model too so we can do like `TwitchPollEventModel.fromDocumentSnapshot(DocumentSnapshot<...>)` instead of parsing it all here.